src/Doctrine/Orm/Extension/ParameterExtension.php

```diff
@@ -71,4 +83,26 @@ public function applyToItem(QueryBuilder $queryBuilder, QueryNameGeneratorInterf
     {
         $this->applyFilter($queryBuilder, $queryNameGenerator, $resourceClass, $operation, $context);
     }
+
+    /**
+     * @param $values array<string, string>
+     *
+     * @throws ContainerExceptionInterface
+     * @throws NotFoundExceptionInterface
+     */
+    private function getFilter(AbstractFilter|string|null $filterId, array $values = []): ?FilterInterface
+    {
+        if ($filterId instanceof AbstractFilter) {
+            $filterId->setManagerRegistry($this->managerRegistry);
+            $filterId->setProperties($values);
```

Comment:
same as for the ODM

Reply:
https://github.com/api-platform/core/pull/6749#discussion_r1824635345